apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/regler/BeregnTrygdetid.kt

```diff
@@ -10,40 +10,59 @@ import no.nav.etterlatte.libs.regler.finnFaktumIGrunnlag
 import no.nav.etterlatte.libs.regler.med
 import no.nav.etterlatte.libs.regler.og
 import java.time.LocalDate
+import java.time.Month
+import java.time.MonthDay
 import java.time.Period
 
 // TODO dato settes riktig senere
 val TRYGDETID_DATO: LocalDate = LocalDate.of(1900, 1, 1)
 
+data class TrygdetidPeriodMedPoengAar(
+    val fra: LocalDate,
+    val til: LocalDate,
+    val poengInnAar: Boolean,
+    val poengUtAar: Boolean
+)
+
 data class TrygdetidPeriodeGrunnlag(
-    val periodeFra: FaktumNode<LocalDate>,
-    val periodeTil: FaktumNode<LocalDate>
+    val periode: FaktumNode<TrygdetidPeriodMedPoengAar>
 )
 
+private fun TrygdetidPeriodMedPoengAar.erEnkelPoengAar() = fra.year == til.year && (poengInnAar || poengUtAar)
+
+private fun TrygdetidPeriodMedPoengAar.poengJustertFra() = if (poengInnAar) {
```

Comment:
Det er kanskje litt "juks" å trekke logikken ut fra en regel som skal bli regelspesifisert isolert - men enig i at det blir lesbart 👍 

Har du noen tanker @henrikg ?

Reply:
Gjorde det bare for lesbarheten.

Vi kan scope dem til kun denne metoden med å flytte dem inn i blokken?

Reply:
Pushet en oppdatering med dette gjort så at det er lettere å se hva jeg mente

Reply:
Det er fint å inkludere den logikken i regelen, så det er nok en grei løsning 👍